### ISSUE TITLE
Blue-Green Deployment - Swap - stopped working

### ISSUE THREAD

**Issue by bjarnegram** [joel74/POSH-LTM-Rest#208]
I'm getting a
The specified LTM name f5.fqdn.name is not valid
error after a call to  https://f5.fqdn.name/mgmt/shared/authn/login 
in a release in Azure Devops.

This worked until some time between February 12th and 19th.

Searching for this error in the repo points me toward token handling.

Could this be related to: 
https://devcentral.f5.com/s/articles/iControl-REST-Authentication-Token-Management

Or do you have any tips to get the "Blue-Green Deployment - Swap" task to work again?

**Comment by joel74** [comment]
Hi, @bjarnegram. Thanks for opening the issue. I'm not familiar with the Blue-Green Deployment - Swap task. Is that something specific to Azure? 

Re: the tokens, you get a new one whenever you create a new session to the F5. If you call New-F5Session, do you get a session object back, and what is the value in the WebSession property of that object? Nothing's changed with the module in about 10 months.

Cheers.

**Comment by bjarnegram** [comment]
Hi!
Sorry, I forgot to mention that this Azure DevOps module is from Jason Vercellones fork: https://github.com/vercellone/POSH-LTM-Rest
I tried to do this in PowerShell and get the same error.
I added some debugging into New-F5Session and got the contents of $LTMError[0]:
"System.Management.Automation.CmdletInvocationException: The underlying connection was closed: An unexpected error occurred on a send. ---> System.Net.WebException: The underlying connection was closed: An unexpected error occurred on a send. ---> System.IO.IOException: Authentication failed because the remote party has closed the transport stream."
Some Googling got me to add the following to New-F5Session (as I've had TLS issues previously):
"[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls -bor [Net.SecurityProtocolType]::Tls11 -bor [Net.SecurityProtocolType]::Tls12"
This did the trick for me.
